specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml

```diff
@@ -17,22 +13,22 @@ options:
     emitter-output-dir: "{project-root}/.."
     examples-directory: "./examples"
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/widgets.json"
-  "@azure-tools/typespec-python":
-    emitter-output-dir: "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
+  "@azure-tools/typespec-python":    
+    package-dir: "azure-contoso-widgetmanager"
     package-mode: "dataplane"
-    package-name: "azure-contoso-widgetmanager"
+    package-name: "{package-dir}"    
   "@azure-tools/typespec-csharp":
+    package-dir: "Azure.Contoso.WidgetManager"
     clear-output-folder: true
-    emitter-output-dir: "{csharp-sdk-folder}/sdk/{service-directory-name}/{namespace}/src"
     model-namespace: false
-    namespace: Azure.Contoso.WidgetManager
+    namespace: "{package-dir}"    
   "@azure-tools/typespec-ts":
-    emitter-output-dir: "{js-sdk-folder}/sdk/{service-directory-name}/contosowidgetmanager-rest"
+    package-dir: "contosowidgetmanager-rest"
     generateMetadata: true
     generateTest: true
     packageDetails:
```

Comment:
@joheredi @MaryGao did we figure out if we can eliminate some of these configurations with smart defaults?

Reply:
I think we could remove below config:

```
generateMetadata: true
generateTest: true
packageDetails:
    description: "Contoso WidgetManager Service"
```

Reply:
Removed.